File: aula-28-02/cliente-servidor-js/servidor.js
// Importa o módulo 'express' para criar o servidor web
const express = require('express');

// Cria uma instância do aplicativo Express
const app = express();

// Rota principal: define o comportamento quando a raiz ("/") for acessada
app.get('/', (req, res) => {
    // Responde com a mensagem 'Hello, Express!' para a requisição
    res.send('Hello, Express!');
});

// Define a porta em que o servidor vai rodar
const PORT = 5000;

// Inicia o servidor e fica ouvindo na porta especificada
app.listen(PORT, () => {
    // Exibe uma mensagem no console quando o servidor estiver rodando
    console.log(`Servidor rodando em http://127.0.0.1:${PORT}`);
});

File: aula-28-02/cliente-servidor-python/servidor.py
# Importa a classe Flask do módulo flask
from flask import Flask

# Cria uma instância do aplicativo Flask
app = Flask(__name__)

# Define a rota para a raiz ("/") do servidor
@app.route("/")
def home():
    # Retorna a mensagem 'Hello, Flask!' quando a rota principal é acessada
    return "Hello, Flask!"

# Verifica se o script está sendo executado diretamente (não importado)
if __name__ == "__main__":
    # Inicia o servidor Flask com a opção de debug ativada
    app.run(debug=True)
